CHANGES/8481.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed the incorrect rejection of ``ws://`` and ``wss://`` urls
```

Comment:
It'd be useful to improve this text with context as requested earlier in https://github.com/aio-libs/aiohttp/pull/8482#discussion_r1666810956

Reply:
Sorry I missed the there was more to that request.

Reply:
Yeah, I only got to take a look again and realized that this description is rather vague. It refers to an incorrect behavior without really getting into what's incorrect about it and where/how it's happening, what's the visible effect for the end-users.

I know this is hard to write and hard to explain, so I'm thinking of ways to lint it better. I was hoping to try integrating https://vale.sh and see if that would help people / give better hints on how to address such things.

Reply:
https://github.com/aio-libs/aiohttp/pull/8513

Reply:
I thought the "incorrect" behaviour referenced here is not in any release of aiohttp, so probably a changelog entry is unneeded anyway.

Reply:
@Dreamsorcerer yep! I actually brought it up in the follow-up and we dropped this note in favor of linking the original one and crediting the contribution to more people, mentioning more PRs/issues.